src/XRP/reliable-submission-xrp-client.ts

```diff
@@ -54,7 +54,7 @@ class ReliableSubmissionXRPClient implements XRPClientDecorator {
     }
 
     // Retrieve the latest ledger index.
-    let latestLedgerSequence = await this.getLastValidatedLedgerSequence()
+    let latestLedgerSequence = await this.getCurrentOpenLedgerSequence()
```

Comment:
random q, as i don't know enough about xrpl - is this necessarily the most recent ledger sequence overall? mostly curious of the var should be renamed as well

Reply:
Nah, we should probably rename this. Technically this code is just plain wrong now, and it's corrected in #459.